Request: additional setup documentation for AWS Amazon Linux 2

Request: additional setup documentation for AWS Amazon Linux 2.  

So far I've figured out that It's necessary to run these, at least if you have not brought in "everything" using sudo yum groupinstall "Development Tools"

sudo yum install openssl-devel
sudo yum install libcurl-devel
sudo yum install gcc
sudo yum install gcc-c++
sudo yum install cmake3

cmake3 ..   -Dssl-path=/usr/ -Dcurl-path=/usr/

```
-- makecerts.sh:
/var/www/phpfiles/osslsigncode-2.3/build/Testing/certs/cert.der: No such file or directory
CMake Error at cmake/CMakeTest.cmake:37 (string):
  string sub-command SUBSTRING requires four arguments.
Call Stack (most recent call first):
  CMakeLists.txt:49 (include)
```

--- a/README.md
+++ b/README.md
@@ -39,9 +39,12 @@ We highly recommend downloading a [release tarball](https://github.com/mtrojnar/
 ```
 * Install prerequisites on macOS with Homebrew:
 ```
-  brew install pkg-config openssl@1.1
+  brew install cmake pkg-config openssl@1.1
   export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
 ```
+**NOTE:** osslsigncode requires CMake 3.6 or newer.
+
+You may need to use `cmake3` instead of `cmake` to complete the following steps on your system.
 * Navigate to the build directory and run CMake to configure the osslsigncode project
   and generate a native build system:
 ```

